rs/state_manager/src/manifest/split/tests.rs

```diff
@@ -400,19 +399,19 @@ fn expected_subnet_1_system_metadata() -> (FileInfo, ChunkInfo) {
     (
         FileInfo {
             relative_path: PathBuf::from(SYSTEM_METADATA_FILE),
-            size_bytes: 63,
+            size_bytes: 65,
             hash: [
-                122, 238, 38, 137, 170, 83, 240, 133, 62, 48, 18, 112, 233, 148, 191, 115, 239,
-                115, 135, 234, 25, 157, 24, 45, 161, 179, 219, 112, 242, 95, 10, 217,
+                135, 141, 106, 99, 2, 246, 244, 239, 255, 254, 121, 198, 217, 0, 168, 217, 254,
+                105, 171, 56, 159, 169, 160, 159, 87, 16, 106, 9, 168, 21, 6, 246,
             ],
         },
         ChunkInfo {
             file_index: 13,
-            size_bytes: 63,
+            size_bytes: 65,
             offset: 0,
             hash: [
-                167, 162, 133, 251, 148, 255, 80, 204, 229, 63, 140, 219, 43, 228, 234, 250, 69,
-                49, 7, 200, 173, 216, 136, 186, 183, 255, 101, 117, 229, 161, 238, 156,
+                48, 206, 112, 95, 239, 131, 62, 43, 197, 160, 224, 171, 236, 107, 12, 29, 127, 75,
+                227, 118, 27, 199, 106, 203, 192, 91, 53, 67, 219, 53, 31, 173,
```

Comment:
It's kind of annoying that these hashes will now change with every new `CURRENT_CERTIFICATION_VERSION`, but I don't see a way around it.

At the very least, I would add a comment (wherever it is that these tests fail with a mismatching hash) explaining why the test fails every time `CURRENT_CERTIFICATION_VERSION` is bumped (because the version is encoded into `SystemMetadata`) and how to fix it (update these hashes).

Reply:
I put a comment there. The other place where things must be adapted is the test with a hash for each version that must be extended. But this was the case before as well. I suppose the only change now is that hashes must be removed if the minimum version is bumped.

Reply:
I suppose we could also use `MIN_SUPPORTED_CERTIFICATION_VERSION`. Then we only have to change them when we bump that.